packages/sdks/core-js-sdk/src/sdk/types.ts

```diff
@@ -64,7 +64,7 @@ export type UserResponse = User & {
   picture?: string;
   roleNames?: string[];
   userTenants?: UserTenant[];
-  createTime: number;
+  createdTime: number;
```

Comment:
This might break compatibility, so maybe we need to support both?

Reply:
And mark the original one as deprecated

Reply:
we don't support `createTime`, its merely a typo on the original PR